### PR TITLE
docs: Improvements to CONTRIBUTING

### PR DESCRIPTION
**Pull request by wkillerud**
* Added a table of contents
* Changed some heading levels to associate the sections with Som
utvikler
* Added a section about Github and the workflow
* Added a section Som designer with a call for contributions

### CODE REVIEW

**Comment by wkillerud** [comment]
Click "View" in the top-right in the Files changed tab to see how the document looks :) Inspired by Debugger.html https://github.com/devtools-html/debugger.html/blob/master/CONTRIBUTING.md

**Review by antidecaf — DISMISSED** [review]
Veldig bra!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,76 @@
...
+issue her på Github med `question`-labelen. Om du heller vil kan du også spørre på den interne HipChat-kanalen
+"Designsystem".
+
+* [Hvordan bidrar jeg?](#hvordan-bidrar-jeg)
+  * [Som designer](#som-designer)
+  * [Som utvikler](#som-utvikler)
+    * [Github-konto og arbeidsflyt](#github-konto-og-arbeidsflyt)
+    * [Utviklingsmiljø](#utviklingsmiljø)
+    * [Kodestandard](#kodestandard)
+    * [Formattering](#formattering)
```
Comment (selbekk) [inline CONTRIBUTING.md]: Formatering* (i know, it looks weird)

**Review by selbekk — DISMISSED** [review]
Great work!

Should we add a note to English-speakers, just informing them about the language choice?

**Comment by wkillerud** [comment]
@selbekk re English, issue #19 was opened to discuss the language choice for these documents. I think we should come to a conclusion there before adding stuff about language to Contributing :)

**Comment by hdaljord** [comment]
Kjempebra!

**Pull request merged**